plugins/woocommerce/src/Blocks/InteractivityComponents/Dropdown.php

```diff
@@ -18,60 +18,96 @@ public static function render( $props ) {
-
 		$action = $props['action'] ?? '';
 
-		// Items should be an array of objects with a label and value property.
-		$items = $props['items'] ?? array();
-
 		$namespace = wp_json_encode( array( 'namespace' => 'woocommerce/interactivity-dropdown' ) );
 
+		$wrapper_class = 'multiple' === $select_type ? '' : 'single-selection';
+
 		ob_start();
 		?>
 		<div data-wc-interactive='<?php echo esc_attr( $namespace ); ?>'>
 			<div class="wc-interactivity-dropdown" data-wc-context='<?php echo esc_attr( wp_json_encode( $dropdown_context ) ); ?>' >
-				<div class="wc-blocks-components-form-token-field-wrapper single-selection" >
+				<div class="wc-blocks-components-form-token-field-wrapper <?php echo esc_attr( $wrapper_class ); ?>" >
 					<div class="components-form-token-field" tabindex="-1">
-						<div class="components-form-token-field__input-container" 
-									data-wc-class--is-active="context.isOpen" 
-									tabindex="-1" 
-									data-wc-on--click="actions.toggleIsOpen" 
-							>
-							<input id="components-form-token-input-1" type="text" autocomplete="off" data-wc-bind--placeholder="state.placeholderText" class="components-form-token-field__input" role="combobox" aria-expanded="false" aria-autocomplete="list" aria-describedby="components-form-token-suggestions-howto-1" value="">
-							<ul hidden data-wc-bind--hidden="!context.isOpen" class="components-form-token-field__suggestions-list" id="components-form-token-suggestions-1" role="listbox">
+						<div class="components-form-token-field__input-container"
+							data-wc-class--is-active="context.isOpen"
+							tabindex="-1"
+							data-wc-on--click="actions.toggleIsOpen"
+						>
+							<?php if ( 'multiple' === $select_type ) { ?>
+								<template
+									data-wc-each="context.selectedItems"
+									data-wc-each-key="context.item.value"
+								>
+									<span class="components-form-token-field__token">
+										<span
+											class="components-form-token-field__token-text"
+											data-wc-text="context.item.label"
+										></span>
+										<button
+											type="button"
+											data-wc-on--click="actions.unselectDropdownItem"
+											data-wc-on--click--parent-action="<?php echo esc_attr( $action ); ?>"
+											class="components-button components-form-token-field__remove-token has-icon"
+										>
+											<svg width="24" height="24" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24" aria-hidden="true" focusable="false">
+												<path d="M12 13.06l3.712 3.713 1.061-1.06L13.061 12l3.712-3.712-1.06-1.06L12 10.938 8.288 7.227l-1.061 1.06L10.939 12l-3.712 3.712 1.06 1.061L12 13.061z"></path>
+											</svg>
+										</button>
+									</span>
+								</template>
+								<?php foreach ( $selected_items as $selected ) { ?>
+									<span
+										class="components-form-token-field__token"
+										data-wc-key="<?php echo esc_attr( $selected['label'] ); ?>"
```

Comment:
I missed this one when I re-reviewed this PR. We're using `label` here but in the template above, `value` is used, it doesn't affect the frontend at all but I think we should still keep it consistent.